src/components/getArticles.jsx

```diff
@@ -0,0 +1,11 @@
+import axios from 'axios'
+
+function getArticles(setItem) {
```

Comment:
this should go into a utils folder as a  .js file, but well done for extracting it out to its own file!